runbot_gitlab_minimal/models/runbot_repo.py

```diff
@@ -113,6 +116,13 @@ def github(self, cr, uid, ids, url, payload=None, ignore_errors=False,
                         json = json[0]
                         json['head'] = {'ref': json['target_branch']}
                         json['base'] = {'ref': json['source_branch']}
+                    if '/commits/' in url:
+                        url = _get_url('/users?search=%s' %
+                                       json['committer_email'], repo.base)
```

Comment:
I have 2 email registered
![image](https://cloud.githubusercontent.com/assets/1387970/24164567/8b309c6c-0e44-11e7-89d5-63525542aa4f.png)
And all good